e2e/specs/solana/create-solana-account.spec.js

```diff
@@ -0,0 +1,157 @@
+import { getFixturesServerPort } from '../../fixtures/utils';
+import SolanaNewFeatureSheet from '../../pages/wallet/SolanaNewFeatureSheet';
+import AddNewHdAccountComponent from '../../pages/wallet/MultiSrp/AddAccountToSrp/AddNewHdAccountComponent';
+import EditAccountNameView from '../../pages/wallet/EditAccountNameView';
+import { SolanaNewFeatureSheetSelectorsIDs } from '../../selectors/wallet/SolanaNewFeatureSheet.selectors';
+
+const ACCOUNT_ONE_TEXT = 'Solana Account 1';
+const ACCOUNT_TWO_TEXT = 'Solana Account 2';
+const NEW_ACCOUNT_NAME = 'Solana Account New Name';
+const PASSWORD = '123123123';
+
+const fixtureServer = new FixtureServer();
+
+describe(SmokeNetworkExpansion('Create Solana account'), () => {
+  beforeAll(async () => {
+    jest.setTimeout(10000);
+    await TestHelpers.reverseServerPort();
+
+    await startFixtureServer(fixtureServer);
+    await loadFixture(fixtureServer, {
+      fixture: new FixtureBuilder()
+      .withSolanaFixture()
+      .withAsyncState({
+        '@MetaMask:existingUser': 'true',
+        '@MetaMask:onboardingWizard': 'explored',
+        '@MetaMask:UserTermsAcceptedv1.0': 'true',
+        '@MetaMask:WhatsNewAppVersionSeen': '7.24.3',
+        '@MetaMask:solanaFeatureModalShown': 'false' // Ensure modal is NOT suppressed
+      })
+      .build(),
+    });
+    await TestHelpers.launchApp({
+      launchArgs: { fixtureServerPort: getFixturesServerPort() },
+    });
+    await loginToApp();
+  });
+
+  afterAll(async () => {
+    await stopFixtureServer(fixtureServer);
+  });
+
+  it('should assert the new Solanafeature announcement sheet', async () => {
+    await Assertions.checkIfVisible(
+      SolanaNewFeatureSheetSelectorsIDs.SOLANA_NEW_FEATURE_SHEET,
+    );
+    await Assertions.checkIfVisible(
+      SolanaNewFeatureSheetSelectorsIDs.SOLANA_LEARN_MORE_BUTTON,
+    );
+    await Assertions.checkIfVisible(
+      SolanaNewFeatureSheetSelectorsIDs.SOLANA_ADD_ACCOUNT_BUTTON_IN_SHEET,
+    );
+    await Assertions.checkIfVisible(
+      SolanaNewFeatureSheetSelectorsIDs.SOLANA_CREATE_ACCOUNT_BUTTON,
+    );
+  });
+
+  it('should create Solana account directly from new feature announcement sheet', async () => {
+    await SolanaNewFeatureSheet.tapCreateAccountButton();
+    await AddNewHdAccountComponent.tapConfirm();
+    await WalletView.tapIdenticon();
+    // Check if the Solana account is created
+    await Assertions.checkIfTextRegexExists(ACCOUNT_ONE_TEXT, 1);
+  });
+
+  it('should create another Solana account from the bottom sheet', async () => {
+    await AccountListBottomSheet.tapAddAccountButton();
+    await AddNewHdAccountComponent.assertContainerIsVisible();
+
+    await device.disableSynchronization();
+    await AddAccountBottomSheet.tapAddSolanaAccount();
+    await device.enableSynchronization();
+
+    await AddNewHdAccountComponent.tapConfirm();
+    await Assertions.checkIfTextRegexExists(ACCOUNT_TWO_TEXT, 2);
```

Comment:
`await Assertions. checkIfTextIsDisplayed(ACCOUNT_TWO_TEXT);`

Doesn't work? Same applies for the other areas where you use Regex
```suggestion
    await Assertions. checkIfTextIsDisplayed(ACCOUNT_TWO_TEXT);
```